foodalert/static/foodalert/js/components/form-template.vue

```diff
@@ -66,8 +66,8 @@
                     @stateAction="this.setValue">
                 </labelled-input>
             </form-category>
-            <form-category section-name="Preview">
-                <p v-html="previewText"></p>
+            <form-category section-name="Message Preview">
+                <pre>{{previewText}}</pre>
```

Comment:
WOuld be better to actually write out the html template here and pass the content down via props (or one big object prop)